src/main/java/dk/sdu/se_f22/sortingmodule/infrastructure/domain/SearchQuery.java

```diff
@@ -8,14 +8,14 @@
  */
 public class SearchQuery {
     /**
-     * Paginations information. 2 places. [page, page size]
+     * Pagination information. 2 places. [page, page size]
      */
-    int[] pagination;
+    int[] pagination = {0, 25};
 
     /**
      * List of range filters
      */
-    ArrayList<Object> range;
+    HashMap<Integer, String[]> range;
```

Comment:
Same here, also using `Map` is better practice

Reply:
I think this was done to enable testing, but I have changed the testing, because it should of course be private.